x-pack/plugins/apm/public/components/app/mobile/transaction_overview/transaction_overview_tabs/use_mobile_statistics_fetcher.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { number } from 'joi';
+import { v4 as uuidv4 } from 'uuid';
+import { useApmServiceContext } from '../../../../../context/apm_service/use_apm_service_context';
+import { useFetcher } from '../../../../../hooks/use_fetcher';
+import { isTimeComparison } from '../../../../shared/time_comparison/get_comparison_options';
+
+const INITIAL_STATE_MAIN_STATISTICS = {
+  mainStatistics: [],
+  requestId: undefined,
+  totalItems: number,
```

Comment:
Shouldn't this be `0` instead of `number`?

Reply:
Yes, I guess I thought I was setting an interface :see_no_evil:

Reply:
Also, that was the reason I needed to hack something else in the call, and now it's working! Thanks for the catch